johnwayodi/store-manager
# #161078894 Add Sale Record Details View

#### What does this PR do?
Adds view where the both the Admin and the Store Attendant can view details of an individual Sale Record

#### Description of Task to be completed?
Ensure this view is displayed for a particular sale record if the sale record is selected

#### How should this be manually tested?
1. `git clone https://github.com/johnwayodi/store-manager.git`
2. Ensure you are in the directory where you cloned this repository, **_store-manager_** by default
3. `git checkout ft-sales-record-details-161078894`
4. open **_sale-record-detail.html_** on browser
#### Any background context you want to provide?
This view is to be available to both the Store attendant and the Administrator and will display the details of an individual sale record containing details of the sale record

#### What are the relevant pivotal tracker stories?
#161078894

#### Screenshots (if appropriate)
![sale-record-details](https://user-images.githubusercontent.com/10683903/46793132-a806e480-cd4d-11e8-825e-373249abff37.png)


#### Questions:
Any feedback on what I can improve or change? Drop a comment or suggestion below
